GOVERNANCE.md

```diff
@@ -0,0 +1,216 @@
+
+
+## Community Processes
+The community around RIOT gathers many IoT developers and users from around the world, from the industry, from academia, and hobbyists.
+The RIOT community is open to everyone.
+Our channels to communicate can be found in our [contributing guidelines].
+The community self-organizes using the roles described below.
+
+
+## Roles
+### Contributors
+Contributors are people who contribute their work to RIOT.
+This includes,
+
+- of course, code contributions, but also
+- bug reports,
+- other issue reports,
+- participation in technical as well as non-technical discussions, or
+- organizational considerations.
+
+Code contributions are very welcome.
+In order to streamline and harmonize code quality, contributors must follow the [contributing guidelines].
+
+
+### Maintainers
+
+Among contributors, some have maintainer status, which consists in rights (write access to the [RIOT GitHub repository](https://github.com/RIOT-OS/RIOT/)) and duties.
+The current maintainers can be found in the [maintainers list].
+
+Maintainers are people who care about RIOT and want to help it grow and improve.
+A maintainer is not just someone who can make changes, but someone who has demonstrated their ability to collaborate with the team, get the most knowledgeable people to review code and docs, contribute high-quality code, and follow through to fix issues (in code or tests).
+More on maintaining RIOT can be found in the [maintaining guidelines].
+
+We are constantly looking for more maintainers.
+So if you are up for that, please start (or continue) contributing code and reviews!
+
+To contact maintainers, the best is to interact over actual RIOT code on [GitHub](https://github.com/RIOT-OS/RIOT/pulls).
+
+#### Becoming a Maintainer
+
+Maintainers can propose to give maintainer status to contributors that have been noticed as particularly active in some domain of RIOT.
+The decision to grant this status is then taken via consensus among the maintainers.
+If there is consensus on granting the status to a particular contributor, a maintainer will personally contact this contributor with the proposal, which the contributor can then accept (or turn down).
+
+Maintainers who are selected will be
+
+- invited to the [maintainers GitHub team] by one of the admins of the RIOT project, which grants them the necessary GitHub rights,
+- invited to the maintainer forum group by the forum moderators, which will give them access to the (private) maintainer part of the forum, and
+- invited to the RIOT-maintainer chat room by one of the moderators of that room, for more informal exchanges between maintainers.
+
+#### Removing a Maintainer
+
+Maintainers may resign at any time if they feel that they will not be able to
+continue fulfilling their project duties.
+
+Maintainers may also be removed after being inactive, upon failure to fulfill their
+Maintainer responsibilities, because of violating the Code of Conduct, or for other reasons.
```

Comment:
I don't expect this to become an issue. But maybe others do?

Reply:
It might be a little to obvious but: actively, persistently, intentionally trying to (or succesfully) harm -especially the security or safety of- RIoT-OS might be a good reason to loose maintainer rights (either as contributor or maintainer). 

Removals for "other reasons" need to follow the process of decision making.

Reply:
> It might be a little to obvious but: actively, persistently, intentionally trying to (or succesfully) harm -especially the security or safety of- RIoT-OS might be a good reason to loose maintainer rights (either as contributor or maintainer).

This is already covered by our Code of Conduct. In this case the maintainer would not just loose their maintainer right, but would also be put on hold on the RIOT communication channels.



> maybe we should state: removals for "other reasons" need to follow the process of decision making. (as "other reasons" are an exceptionally low bar)

Maybe we should just remove the “for other reasons” wording altogether?

Reply:
> This is already covered by our Code of Conduct. In this case the maintainer would not just loose their maintainer right, but would also be put on hold on the RIOT communication channels.

Isn't the Code of Conduct about communication rather than the OS (code) itself, I thought this (my suggestion) is more about the OS and Code rather than the communication (more like (try) introducing backdoor/ harm the product rather than the community)

Reply:
Ah. Yes, this makes sense.